dev/reference/configuration/tidb-server/tidb-specific-variables.md

```diff
@@ -381,3 +396,123 @@ Usage example:
+        ```sql
+        tidb >set @@tidb_constraint_check_in_place=1
+        tidb >begin
+        tidb >insert into t values (1);
+        ERROR 1062 : Duplicate entry '1' for key 'PRIMARY'
+        ```
+
+### tidb_check_mb4_value_in_utf8
+
+- Scope: SERVER
+- Default value: 1, indicating check the validity of UTF-8 data. This default behavior is compatible with MySQL.
+- This variable is used to set whether to check the validity of UTF-8 data.
+- To upgrade an earlier version (TiDB v2.1.1 or earlier), you may need to disable this option. Otherwise, you can successfully write invalid strings in an earlier version but fail to do this in a later version, because you do not do data validity check in the earlier version. For details, see [FAQs After Upgrade](/faq/upgrade.md).
+
+### tidb_opt_insubq_to_join_and_agg
+
+- Scope: SESSION | GLOBAL
+- Default value: 1
+- This variable is used to set whether to enable the optimization rule that converts a subquery to join and aggregation.
+- For example, after you enable this optimization rule, the subquery is converted as follows:
+
+    ```sql
+    select * from t where t.a in (select aa from t1)
+    ```
+
+    The subquery is converted to join as follows:
+
+    ```sql
+    select * from t, (select aa from t1 group by aa) tmp_t where t.a = tmp_t.aa
+    ```
+
+    If `t1` is limited to be unique and not null in the `aa` column. You can use the following statement, without aggregation.
+
+    ```sql
+    select * from t, t1 where t.a=t1.a
+    ```
+
+### tidb_opt_correlation_threshold
+
+- Scope: SESSION | GLOBAL
+- Default value: 0.9
+- This variable is used to set the threshold value of estimating the row count of the current column by a method that estimates the number of rows of the `handle` column using the column histogram. If the order correlation between the current column and the `handle` column exceeds the threshold value, this method is enabled.
+
+### tidb_opt_correlation_exp_factor
+
+- Scope: SESSION | GLOBAL
+- Default value: 1
+- When the method that estimates the number of rows of the `handle` column using the column histogram cannot be used, the heuristic estimation method is used. This variable is used to control the behavior of the heuristic method. 
+    - When the value is 0, the heuristic method is not used.
+    - When the value is greater than 0:
+        - A larger value indicates that an index scan will probably be used in the heuristic method.
+        - A smaller value indicates that a table scan will probably be used in the heuristic method.  
+
+### tidb_enable_window_function
+
+- Scope: SESSION | GLOBAL
+- Default value: 1, indicating enabling the window function feature.
+- This variable is used to control whether to enable the support for window functions. Window functions may use reserved or key words. This might cause SQL statements that could be executed normally cannot be parsed after upgrading TiDB. In this case, you can set `tidb_enable_window_function` to `0`.
```

Comment:
```suggestion
- This variable is used to control whether to enable the support for window functions. Note that window functions may use reserved keywords. This might cause SQL statements that could be executed normally cannot be parsed after upgrading TiDB. In this case, you can set `tidb_enable_window_function` to `0`.
```